scripts/generate_dos_questions.sh

```diff
@@ -5,6 +5,9 @@
 # Usage:
 #   FRAMEWORKS_PATH=frameworks/digital-outcomes-and-specialists/questions/services/ ./scripts/generate_dos_questions.sh
 
+>&2 echo "This can no longer be used, it will clobber the questions, soz."
```

Comment:
Why won't this work any more?  Won't we need to update the questions in the supplier pack to show the new South West option?